api/mutations/directMessageThread/archiveDirectMessageThread.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import type { GraphQLContext } from '../../';
+import UserError from '../../utils/UserError';
+import {
+  updateIsDirectMessageThreadArchived,
+  getDirectMessageThread,
+} from '../../models/usersDirectMessageThreads';
+
+type isArchivedDMThreadInput = {
+  input: {
+    isArchived: boolean,
+    threadId: string,
+  },
+};
+
+export default async (
+  _: any,
+  { input }: isArchivedDMThreadInput,
+  { user }: GraphQLContext
+) => {
+  const currentUser = user;
+
+  if (!currentUser) {
+    return new UserError('You must be signed in to send a direct message.');
```

Comment:
There should be a different message right?

Reply:
Oops. Yeah you’re right!